### PR TITLE
Validate sortcode and account number

### PR DESCRIPTION
**Pull request by dishantmehta93**

### CODE REVIEW

**Comment by rasarasw** [comment]
@dheerajarora  - Done, the review. Can you please review and merge the code?

File: src/main/java/uk/co/techblue/postcodeanywhere/resource/BankingResource.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import javax.ws.rs.core.Response;
+
+import uk.co.techblue.postcodeanywhere.Resource;
+
+/**
+ * The Interface BankingResource.
+ *
+ * @author <a href="mailto:dishant.mehta@techblue.co.uk">Dishant Mehta</a>
+ */
+@Path("BankAccountValidation/Interactive")
```
Comment (rasarasw) [inline src/main/java/uk/co/techblue/postcodeanywhere/resource/BankingResource.java]: @dishantmehta93  - Shouldn't we add '/' before the BankAccountValidation?

Comment (rasarasw) [inline src/main/java/uk/co/techblue/postcodeanywhere/resource/BankingResource.java]: @dishantmehta93  - Can you please add '/' before the BankAccountValidation?

File: src/main/java/uk/co/techblue/postcodeanywhere/dto/captureplus/BankingResponse.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import uk.co.techblue.postcodeanywhere.dto.BaseDto;
+
+/**
+ * The Class SortCodeResponse.
+ *
+ * @author <a href="mailto:dishant.mehta@techblue.co.uk">Dishant Mehta</a>
+ */
+public class BankingResponse extends BaseDto {
```
Comment (dheerajarora) [inline src/main/java/uk/co/techblue/postcodeanywhere/dto/captureplus/BankingResponse.java]: Rename the class to BankAccountValidationResponse

File: src/main/java/uk/co/techblue/postcodeanywhere/service/BankingService.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import uk.co.techblue.postcodeanywhere.dto.captureplus.BankingResponse;
+import uk.co.techblue.postcodeanywhere.exception.LookupException;
+import uk.co.techblue.postcodeanywhere.resource.BankingResource;
+
+/**
+ * The Class BankingService.
+ *
+ * @author <a href="mailto:dishant.mehta@techblue.co.uk">Dishant Mehta</a>
+ */
+public class BankingService extends Service<BankingResource> {
```
Comment (dheerajarora) [inline src/main/java/uk/co/techblue/postcodeanywhere/service/BankingService.java]: Rename this to BankAccountValidationService

File: src/main/java/uk/co/techblue/postcodeanywhere/resource/BankingResource.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+public interface BankingResource extends Resource {
```
Comment (dheerajarora) [inline src/main/java/uk/co/techblue/postcodeanywhere/resource/BankingResource.java]: Rename this to BankAccountValidationResource

File: src/main/java/uk/co/techblue/postcodeanywhere/dto/captureplus/BankingValidationRecord.java [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import uk.co.techblue.postcodeanywhere.dto.BaseDto;
+
+/**
+ * The Class SortCodeRecord.
+ *
+ * @author <a href="mailto:dishant.mehta@techblue.co.uk">Dishant Mehta</a>
+ */
+public class BankingValidationRecord extends BaseDto {
```
Comment (dheerajarora) [inline src/main/java/uk/co/techblue/postcodeanywhere/dto/captureplus/BankingValidationRecord.java]: Rename this to BankAccountValidationRecord

**Review by dheerajarora — CHANGES_REQUESTED** [review]
I cannot see the code to test this service in test cases. Please add a test method to test BankAccountValidationService

File: src/main/java/uk/co/techblue/postcodeanywhere/resource/BankingResource.java [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+import uk.co.techblue.postcodeanywhere.Resource;
+
+/**
+ * The Interface BankingResource.
+ *
+ * @author <a href="mailto:dishant.mehta@techblue.co.uk">Dishant Mehta</a>
+ */
+@Path("/BankAccountValidation/Interactive")
+public interface BankingResource extends Resource {
```
Comment (dheerajarora) [inline src/main/java/uk/co/techblue/postcodeanywhere/resource/BankingResource.java]: it needs to be changed to BackAccountValidationResource

**Review by dheerajarora — APPROVED** [review]

**Review by dheerajarora — APPROVED** [review]

**Pull request merged**